tests/linearizability/linearizability_test.go

```diff
@@ -192,3 +185,39 @@ type trafficConfig struct {
 	clientCount int
 	traffic     Traffic
 }
+
+func checkOperationsAndPersistResults(t *testing.T, operations []porcupine.Operation, clus *e2e.EtcdProcessCluster) porcupine.CheckResult {
+	path, err := filepath.Abs(filepath.Join(resultsDirectory, strings.Replace(t.Name(), "/", "_", -1)))
+	if err != nil {
+		t.Error(err)
+	}
+	err = os.MkdirAll(path, 0700)
+	if err != nil {
+		t.Error(err)
```

Comment:
`t.Error(err)` means that the test will continue to execute .

```suggestion
		t.Fatal(err)
```

Reply:
Will still want to report model not linearizable even if writing report will not work.